front/app/components/Form/Components/Controls/MapInput/MapControl.tsx

```diff
@@ -110,7 +110,7 @@ const MapControl = ({ ...props }: ControlProps) => {
   };
 
   return (
-    <>
+    <Box key={`${path}`}>
       <Box>
```

Comment:
Issue seemed to be caused by rendering problems when a new page was loaded with more mapping questions. Adding the key here makes sure we refresh/rerender the controls properly.

Reply:
If you can think of any issues with this approach though that I haven't considered, just let me know!

Reply:
The odd thing is this never used to happen with these questions :thinking: Given that it's happening [for other custom field types as well](https://go-vocal.slack.com/archives/C06CJ4D3B0R/p1740518684942119?thread_ts=1740467651.849009&cid=C06CJ4D3B0R), I wonder if there's another cause :thinking: Adrien's quick-fix for the survey page itself will fix this, so I'll close this PR at least :)